.github/workflows/ci.yml

```
@@ -0,0 +1,119 @@
+name: Integration
+on:
+  pull_request:
+  push:
+    branches: [ master, develop ]
```

```suggestion
    branches: [ master ]
```
No develop branch here.